reference_validator/reference_validator.py

```diff
@@ -1,786 +1,42 @@
-                flag = False
-                for res in [x for x in node.resources if x.type.endswith('.yaml')]:
-                    for prop, value in six.iteritems(res.properties):
-                        if value == False:
-                            flag = True
-                            break
-                    if flag:
-                        break
-                if flag:
-                    node.ok = False
-                    if self.pretty_format:
-                        print(YAML_colours.BOLD + 'Properties without corresponding parameter:' +
-                            YAML_colours.DEFAULT)
-                    else:
-                        print('Properties without corresponding parameter :')
-
-                    for res in [x for x in node.resources if x.type.endswith('.yaml')]:
-                        for prop, value in res.properties.iteritems():
-                            if value == False:
-                                if self.pretty_format:
-                                    print('- ' + YAML_colours.YELLOW + prop + YAML_colours.DEFAULT +
-                                          ' in ' + YAML_colours.YELLOW + res.name +
-                                          YAML_colours.DEFAULT)
-                                else:
-                                    print('- ' + prop + ' in ' + res.name)
-                    print('')
-
-                # Print unused resources (optional)
-                if (self.print_unused) and [True for x in node.resources if not x.used]:
-                    if (self.pretty_format):
-                        print(YAML_colours.BOLD + 'Resources without reference:' +
-                              YAML_colours.DEFAULT)
-                    else:
-                        print('Resources without reference:')
-
-                    for resource in node.resources:
-                        if resource.used == False:
-                            if self.pretty_format:
-                                print('- ' + YAML_colours.YELLOW + resource.name + YAML_colours.DEFAULT +
-                                      ' (' + resource.type + ')')
-                            else:
-                                print('- ' + resource.name)
-                    print('')
-
-                # Print file status as OK if there were no problems
-                if node.ok:
-                    if self.pretty_format:
-                        print(YAML_colours.BOLD + 'Status: ' + YAML_colours.GREEN + 'OK' +
-                              YAML_colours.DEFAULT)
-                    else:
-                        print ('Status: OK')
-                else:
-                    if self.pretty_format:
-                        print(YAML_colours.BOLD + 'Status: ' + YAML_colours.RED + 'FAILED' +
-                              YAML_colours.DEFAULT)
-                    else:
-                        print('Status: FAILED')
-
-                print('\n\n')
 
+import YAML_HotValidator as VALIDATOR
```

Comment:
By convention, uppercase names are reserved for constants, not modules. Could this be `validator` instead?